mappings/net/minecraft/nbt/AbstractNbtList.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_2483 net/minecraft/nbt/AbstractNbtList
+	COMMENT Represents an abstraction of a mutable NBT list which holds elements of the same type.
+	METHOD add (ILjava/lang/Object;)V
+		ARG 1 index
+		ARG 2 value
+	METHOD method_10533 add (ILnet/minecraft/class_2520;)Z
+		ARG 1 index
+		ARG 2 element
+	METHOD method_10535 set (ILnet/minecraft/class_2520;)Z
```

Comment:
Sorry, my bad, but 10533 and 10535 are not list methods but distinct methods for other purposes. Maybe restore their previous `addElement` `setElement` names or you can find better names

Reply:
I honestly don't have better names so I'm gonna just restore the previous names.